views/fixed_no_toolbar.erb

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
Every time you reload the check_in or check_in_login pages a link that says "close" is flashed really quickly and disappears. I think it has something to do with this file maybe there is some code that could be removed from here to prevent the bug.